feedback/feedback.py

```diff
@@ -213,15 +216,15 @@ def get_url(icon_type, i):
             styling. For example, at the time of this writing, we do
             selected through CSS, rather than by using those icons.
             '''
-            templates = {'inactive': 'public/default_icons/i{set}{i}.png',
-                         'active': 'public/default_icons/a{set}{i}.png',
-                         'selected': 'public/default_icons/s{set}{i}.png'}
```

Comment:
Why are we removing the selected states?
is it same as pressed states?

Reply:
Selected isn't used anywhere in the state of the XBlock. I think it was supposed to be like in the hover state, but it was never implemented. (just a supposition)